test/controllers/reviews_controller_test.rb

```diff
@@ -0,0 +1,17 @@
+require 'test_helper'
+
+class ReviewsControllerTest < ActionController::TestCase
+  # test "the truth" do
```

Comment:
This test probably ran into trouble because you have to assert something to be true. Asserts have to compare 2 things, not just one thing. (I believe)